distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso

```diff
@@ -411,6 +411,8 @@ type Column
              example_lt = Examples.integer_column < 1
     < : Column | Any -> Column
     < self other = Value_Type.expect_comparable self other <|
+        if other.is_a Column then
+            IO.println "AAA "+self.value_type.to_text+" :: "+other.value_type.to_text
```

Comment:
looks like leftover

Reply:
Sorry -- these changes should not have been on this branch, it was a mistake.